kuma/core/tests/test_helpers.py

```diff
@@ -167,6 +168,21 @@ def test_unknown_format(self):
         with pytest.raises(DateTimeFormatError):
             datetimeformat(self.context, date_today, format='unknown')
 
+    @mock.patch('babel.dates.format_datetime')
+    def test_broken_format(self, mocked_format_datetime):
+        value_test = datetime.fromordinal(733900)
+        value_english = format_datetime(value_test, locale=u'en_US')
+        self.context['request'].LANGUAGE_CODE = u'fr'
+        mocked_format_datetime.side_effect = [
+            # first call is returning a KeyError as if the format is broken
+            KeyError,
+            # second call returns the English fallback version as expected
+            value_english,
+        ]
+        value_returned = datetimeformat(self.context, value_test,
+                                        format='datetime')
```

Comment:
Why not have it output json and then you get a more "raw" value that you can check more easily?